### ISSUE TITLE
More documentation, examples for remote parameters

### ISSUE THREAD

**Issue by mbechard** [disguise-one/RenderStream#31]
Although I can make assumption on how this works. It'd be good if getting remote parameters had more documentation. If I have a mix of numbers, poses, images, and text, how are they retrieved?
I assume I separate the types and collapse the indices down. For number/poses I assume they are tightly packed so a float for a number followed by 16 floats for a pose followed by another float for a number.
Is that correct?
Are the indices for the rs_getFrameText() collapsed down or are those the index of the text parameter in the whole remote parameter list?

Thanks

**Comment by husberg-disguise** [comment]
Yes, separated by type and collapsed. Currently the UE plugin source is the best reference: https://github.com/disguise-one/RenderStream-UE/blob/r1.29_UE4.27/Source/RenderStream/Private/RenderStreamSceneSelector.cpp#L296

We're internally debating whether to change the API to be clearer or write helper C++ classes to hide the complexity, but we'll hopefully have something before the next release.

**Comment by twhittock** [comment]
The RenderStream-py repo has https://github.com/disguise-one/RenderStream-py/blob/11ab345cc3a9adfa953176532907d60528c8da3b/src/renderstream/renderstream.py#L700 which is a high level example of how to get all frame parameters. We'll be adding documentation and a C++ wrapper in this repository soon with a similar implementation so it's less opaque.

**Comment by twhittock** [comment]
https://github.com/disguise-one/RenderStream/blob/r1.30/src/include/renderstream.hpp has a rough and ready C++ wrapper which shows the behaviour. Will add documentation to describe the process too.
